### DISCUSSION TITLE
Is there any way to get dark mode on linux?

### DISCUSSION THREAD

**Discussion by quesada** [zim-desktop-wiki/zim-desktop-wiki#1968] [Q&A] [answered]
I know zim is gtk, I'm on a kde desktop and while FF (also gtk) gets the dark mode from the OS settings, zim doesn't. Anything I could configure?
I'm using openbox as my wm, not kwin.

**Comment by dehesselle** [comment, marked as answer]
There is a setting `GTK_THEME` for that. It can be set as environment var or in a configuration file. See here for getting started: https://unix.stackexchange.com/questions/14129/gtk-enable-set-dark-theme-on-a-per-application-basis

So you're still a Zim user after all? I just read your response in another discussion stating that you've moved on to another app.

**Reply by introt** [reply]
Related dark mode stuff: #1727,  #1372 contains step-by-step instructions
